esmvalcore/preprocessor/__init__.py

```diff
@@ -158,7 +157,10 @@
     'timeseries_filter',
     'linear_trend',
     'linear_trend_stderr',
+    # Convert units
     'convert_units',
```

Comment:
wait - shouldn't be convert units last?

Reply:
computationally it makes sense it to be last, since you'd convert units only of a single multimodel cube no?

Reply:
What if you want to compute both the multimodel mean and variance? They wouldn't have the same unit so you couldn't use convert_units.

Reply:
> computationally it makes sense it to be last, since you'd convert units only of a single multimodel cube no?

No, because the multimodel statistics function also keeps all the input cubes, doing it after the multimodel statistics will mean that you have to convert the units of the number of input cubes + the number of multimodel statistics, whereas if you do it before you only need to convert the input cubes.

Reply:
solid points, ignore me comment, man :+1: :beer: